paymentsheet-example/src/androidTest/java/com/stripe/android/utils/LpmRepositoryKtx.kt

```diff
@@ -2,27 +2,33 @@ package com.stripe.android.utils
 
 import android.content.Context
 import com.stripe.android.lpmfoundations.luxe.LpmRepository
+import com.stripe.android.lpmfoundations.paymentmethod.PaymentMethodMetadata
 import com.stripe.android.model.PaymentMethod
 import com.stripe.android.testing.PaymentIntentFactory
+import com.stripe.android.ui.core.BillingDetailsCollectionConfiguration
 
 internal fun initializedLpmRepository(context: Context): LpmRepository {
     val repository = LpmRepository(
         LpmRepository.LpmRepositoryArguments(
             resources = context.resources,
-            isFinancialConnectionsAvailable = { true }
         )
     )
 
     repository.update(
-        stripeIntent = PaymentIntentFactory.create(
-            paymentMethodTypes = PaymentMethod.Type.values().map { it.code },
-            paymentMethodOptionsJsonString = """
+        metadata = PaymentMethodMetadata(
```

Comment:
My hope is to move these end to end tests away from using the specs directly. I'm hoping that'll be possible in a follow up.